Examples/security_rules.ruleset

```diff
@@ -0,0 +1,135 @@
+let backup_retention = 7
+let allowed_ports = [80, 443]
+
+AWS::AmazonMQ::Broker AutoMinorVersionUpgrade == true << Version upgrades should be enabled to receive security updates
+AWS::AmazonMQ::Broker EncryptionOptions.UseAwsOwnedKey == false << CMKs should be used instead of AWS-provided KMS keys
+AWS::AmazonMQ::Broker EngineVersion == 5.15.10 << Broker engine version should be at least 5.15.10
+AWS::AmazonMQ::Broker Logs.Audit == true << Audit logging should be enabled
+AWS::AmazonMQ::Broker Logs.General == true << General logging should be enabled
+AWS::AmazonMQ::Broker PubliclyAccessible == false << Broker should not be publicly accessible
+
+AWS::AmazonMQ::Configuration EngineVersion == 5.15.10 << Broker engine version should be at least 5.15.10
+
+AWS::ApiGateway::DomainName SecurityPolicy == TLS_1_2 << API Gateway DomainName SecurityPolicy should use TLS 1.2
+
+AWS::CloudFront::Distribution ViewerCertificate.MinimumProtocolVersion == TLSv1.2_2018 << CloudFront TLS version should be 1.2
+
+AWS::CloudTrail::Trail EnableLogFileValidation == true << CloudTrail file validation should be enabled
+AWS::CloudTrail::Trail IncludeGlobalServiceEvents == true << CloudTrail should include global service events
+AWS::CloudTrail::Trail IsLogging == true << CloudTrail logging should be enabled
+AWS::CloudTrail::Trail IsMultiRegionTrail == true << CloudTrail trails should cover all regions
+
+AWS::CodeBuild::Project EncryptionKey == /.*/ << CodeBuild should use a CMK instead of AWS-provided KMS keys
+AWS::CodeBuild::Project VpcConfig.VpcId == /.*/ << CodeBuild projects should be built in a VPC
+
+AWS::CodeBuild::SourceCredential Token == /^\{\{resolve:secretsmanager.*$/ << CodeBuild credentials should use a reference instead of plaintext
```

Comment:
related discussions of sensitive properties that shouldn't be hardcoded:
https://github.com/aws-cloudformation/cfn-python-lint/pull/1181
https://github.com/stelligent/cfn_nag/issues/253